packages/payload/src/uploads/types.ts

```diff
@@ -12,6 +12,14 @@ export type FileSize = {
   width: null | number
 }
 
+/**
+ * FileSize_P4 is a more precise type, and will replace FileSize in Payload v4.
+ * I am not encouraging users to use this type via tsdoc yet, in case we find more breaking changes to do prior to v4.
```

Comment:
Should add an `@internal` jsdoc note if this is the case.

Reply:
It's not actually needed because it's not exported by the package (UploadData is exported in `packages/richtext-lexical/src/index.ts`). I'm adding it anyway, it doesn't hurt 😁